webapp/templates/404.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
Could we also render this via react? e.g when server detects a 404 we render "index" page, but it shows a 404 because no route is found?

Reply:
done